scripts/pr-convergence.py

```
@@ -0,0 +1,971 @@
+            "enabled": False,
+            "previous_snapshot_found": False,
+            "changed": False,
+            "categories": {},
+        }
+
+    categories = {
+        "comments": changed_keys(previous.get("comments", []), current.get("comments", [])),
+        "reviews": changed_keys(previous.get("reviews", []), current.get("reviews", [])),
+        "checks": changed_keys(previous.get("checks", []), current.get("checks", []), "name"),
+        "commits": sorted(
+            set(str(item) for item in current.get("commits", []))
```

Ruff (0.16.0)

[warning] Unnecessary generator (rewrite as a set comprehension)

Rewrite as a set comprehension

(C401)